tests/scenario/test_dvt_scenario.py

```diff
@@ -0,0 +1,376 @@
+    new_reward_address = "0x000000000000000000000000000000000000dEaD"
+    set_node_operator_reward_address_calldata = (
+        "0x" + encode_single("((uint256,address)[])", [[(6, new_reward_address)]]).hex()
+    )
+
+    easytrack_executor(
+        commitee_multisig,
+        set_node_operator_reward_address_factory,
+        set_node_operator_reward_address_calldata,
+    )
+
+    cluster = simple_dvt.getNodeOperator(6, True)
+    assert cluster["rewardAddress"] == new_reward_address
+
+    # add signing keys to node operator
+    no_5_id = 5
+    no_5 = simple_dvt.getNodeOperator(no_5_id, False)
+
+    simple_dvt.addSigningKeysOperatorBH(
+        no_5_id,
+        len(signing_keys["pubkeys"]),
+        "0x" + "".join(signing_keys["pubkeys"]),
+        "0x" + "".join(signing_keys["signatures"]),
+        {"from": no_5["rewardAddress"]},
+    )
+
+    assert cluster["totalVettedValidators"] == 0
+
+    # Increase staking limit from NO
+    increase_vetted_validators_limit_calldata = (
+        "0x" + encode_single("(uint256,uint256)", [no_5_id, 1]).hex()
+    )
+
+    easytrack_executor(
+        no_5["rewardAddress"],
+        increase_vetted_validators_limit_factory,
+        increase_vetted_validators_limit_calldata,
+    )
+
+    cluster = simple_dvt.getNodeOperator(no_5_id, False)
+    assert cluster["totalVettedValidators"] == 1
+
+    increase_vetted_validators_limit_calldata = (
+        "0x" + encode_single("(uint256,uint256)", [no_5_id, 2]).hex()
+    )
+
+    easytrack_executor(
+        clusters[no_5_id]["manager"],
+        increase_vetted_validators_limit_factory,
+        increase_vetted_validators_limit_calldata,
+    )
+
+    cluster = simple_dvt.getNodeOperator(no_5_id, False)
+    assert cluster["totalVettedValidators"] == 2
+
+    # Increase staking limit with commitee
+    simple_dvt.addSigningKeysOperatorBH(
+        no_5_id,
+        len(signing_keys["pubkeys"]),
+        "0x" + "".join(signing_keys["pubkeys"]),
+        "0x" + "".join(signing_keys["signatures"]),
+        {"from": no_5["rewardAddress"]},
+    )
+
+    no_6_id = 6
+    no_6 = simple_dvt.getNodeOperator(no_6_id, False)
+    simple_dvt.addSigningKeysOperatorBH(
+        no_6_id,
+        len(signing_keys["pubkeys"]),
+        "0x" + "".join(signing_keys["pubkeys"]),
+        "0x" + "".join(signing_keys["signatures"]),
+        {"from": no_6["rewardAddress"]},
+    )
+
+    set_vetted_validators_limit_calldata = (
+        "0x"
+        + encode_single("((uint256,uint256)[])", [[(no_5_id, 4), (no_6_id, 3)]]).hex()
+    )
+    easytrack_executor(
+        commitee_multisig,
+        set_vetted_validators_limit_factory,
+        set_vetted_validators_limit_calldata,
+    )
+
+    cluster_5 = simple_dvt.getNodeOperator(no_5_id, False)
+    assert cluster_5["totalVettedValidators"] == 4
+    cluster_6 = simple_dvt.getNodeOperator(no_6_id, False)
+    assert cluster_6["totalVettedValidators"] == 3
+
+    # Revoke MANAGE_SIGNING_KEYS role
```

Comment:
For this test case, we also have to check, that voting later may create MANAGE_SIGNING_KEYS permission to make some action assigning it to itself.